spring-ws-core/src/main/java/org/springframework/ws/transport/http/HttpUrlConnection.java

```diff
@@ -118,13 +118,12 @@ protected long getResponseContentLength() throws IOException {
 
 	@Override
 	public Iterator<String> getResponseHeaders(String name) throws IOException {
-		String headerField = connection.getHeaderField(name);
-		if (headerField == null) {
-			return Collections.<String>emptyList().iterator();
-		}
-		else {
-			Set<String> tokens = StringUtils.commaDelimitedListToSet(headerField);
-			return tokens.iterator();
+		Map<String, List<String>> mapHeader = connection.getHeaderFields();
+		List<String> listHeaderValues = mapHeader.get(name);
+		if(null == listHeaderValues) {
```

Comment:
Under the Spring coding conventions, we write this as `if (listHeaderValues == null)`. Note the space after `if` and the flip of the clauses. See https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style for more info.